internal/consts/consts.go

```diff
@@ -82,6 +82,13 @@ const (
 	MountTypeNomad      = "nomad"
 	MountTypeKubernetes = "kubernetes"
 
+	/*
+		Vault version constants
+	*/
+	VaultVersion11 = "1.11.0"
```

Comment:
Suggest making these `version.Version` variables instead, then there is no need to reparse them for every API request.